Nouvelle api

jbrv opened the pull request.

jbrv committed:
Nouvelle API: implémentation

jbrv committed:
Nouvelle API: Documentation

Oui, c'est original, on spécifie et on documente...

Trowa commented on Protocole.md:
```diff
@@ -0,0 +1,148 @@
+
+`h`: Arrêt d'urgence
+
+`r`: Reset de l'arrêt d'urgence
```
Précise que le robot repart pas sans ça suite à un h

Trowa commented on asserv/main/Main.cpp:
```diff
@@ -301,12 +171,13 @@ void Live_isr()
+                (double) odometrie->getTheta(),
+                (int) commandManager->getLastCommandStatus(),
+                (int) motorController->getVitesseG(),
+                (int) motorController->getVitesseD());
```
dièse et juste les valeurs séparées par des ; ça faciliterai le parsing en java

jbrv committed:
Peer review: précisions sur l'arrêt d'urgence

jbrv committed:
Peer review: Facilitation du parsing du statut pour Java

L'IA étant en Java, et Java n'ayant pas jugé utile d'implémenter un truc
aussi basique qu'un scanf() (alors qu'on peut faire des "switch/case" sur des
String... bref!), on sépare tous les champs du statut par un même
caractère.
